tunnel/tunnel.go

```diff
@@ -114,8 +40,8 @@ func StartReverseTunnel(tunnelConfig *Config, wg *sync.WaitGroup) {
 	}
 
 	// This catches CTRL C and closes the ssh
-	c := make(chan os.Signal)
-	signal.Notify(c,
+	s := make(chan os.Signal)
```

Comment:
I think better variable names here would make this non-intuitive shitty syntax at least slightly more bearable.